skyvern/webeye/actions/handler.py

```diff
@@ -643,6 +662,22 @@ def get_select_id_in_label_children(scraped_page: ScrapedPage, element_id: int)
     return None
 
 
+def get_checkbox_id_in_label_children(scraped_page: ScrapedPage, element_id: int) -> int | None:
+    """
+    search checkbox/radio in the children of <label>
+    """
+    LOG.info("Searching checkbox/radio in the label children", element_id=element_id)
+    element = scraped_page.id_to_element_dict.get(element_id, None)
+    if element is None:
+        return None
+
+    for child in element.get("children", []):
+        if child.get("tagName", "") == "input" and child.get("attributes", {}).get("type") in ["checkbox", "radio"]:
```

Comment:
is it possible there are multiple? if so I think we should make a decision but log an error.

Reply:
According to the HTML rule, there shouldn't be multiple `<input>` or `<select>` in the child of label.
Otherwise `<label>` will lose its default binding.